tests/js/utilsTests.js

```diff
@@ -0,0 +1,47 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
```

Comment:
There should also be tests for `gpii.d3.jQueryToD3` and `gpii.d3.addD3Listeners`